packages/core/src/sheets/worksheet.ts

```diff
@@ -86,9 +87,95 @@ export class Worksheet {
         return this._spanModel;
     }
 
+    /**
+     * Get the style of the column.
+     * @param {number} column The column index
+     * @returns {Nullable<IStyleData>|string} The style of the column
+     */
+    getColumnStyle(column: number): Nullable<IStyleData> | string {
```

Comment:
Why are there two methods here? `getColumnStyleInternal` and `getColumnStyle` seem to be essentially the same method. The `Styles.get` method also dynamically determines the type of `s`.

Reply:
it not same , the getColumnStyle will return style id, but the internal one which should not public, it useful in sketon for get style data directly